.github/workflows/ci.yml

```diff
@@ -86,6 +86,24 @@ jobs:
         env:
           HEAD_SHA: ${{ github.event.pull_request.head.sha }}
 
+  webrtc_tests:
+    name: Run all webrtc tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: dtolnay/rust-toolchain@stable
+        with:
+          target: wasm32-unknown-unknown
+
+      - uses: taiki-e/cache-cargo-install-action@v1
+        with:
+          tool: wasm-pack@0.12.0
+
+      - name: Build webrtc-browser example
+        run: |
+          cd examples/webrtc-browser
+          wasm-pack build --target web --out-dir static
```

Comment:
Can you make this part of the examples check please instead of introducing another job?